package.json

```diff
@@ -28,13 +28,13 @@
     "eslint-plugin-react": ">=7.14.2"
   },
   "peerDependencies": {
-    "babel-eslint": ">=8.0.1",
-    "eslint": ">=5.3.0"
+    "@babel/eslint-parser": "^7.11.0",
```

Comment:
How did you pick the differing version numbers for the `peerDependencies` vs. `devDependencies`?

Reply:
The one listed in `devDependencies` is the current latest. The one under `peerDependencies` is the requirement expressed by indirect dependencies (in this case `argparse` is the limiting factor). IMO it's pretty arbitrary; I would be OK with changing the `peerDependencies` entry to `>=7.17.0` or `^7.17.0`.